src/main/scala/latis/input/HapiAdapter.scala

```diff
@@ -4,40 +4,103 @@ import latis.model._
 import latis.util._
 import java.net.URI
 import latis.data.SampledFunction
-import latis.data.HapiFunction
+import latis.ops.Operation
+import latis.ops.Selection
+import latis.ops.Projection
```

Comment:
This import isn't used.

Reply:
Hmm, eclipse is no longer warning me of unused imports. Now if we can figure out how to get IDEs and/or scalafmt to do this for us....